workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -111,14 +115,24 @@ def drift_3day(diags):
     return drift
 
 
-@add_to_metrics("time_and_global_mean_bias")
-def time_mean_bias(diags):
-    global_mean_bias = grab_diag(diags, "mean_bias_physics_global")
+@add_to_metrics("time_and_global_mean_value")
+def time_and_global_mean_value(diags):
+    time_mean_value = grab_diag(diags, "time_mean_value")
+    area = diags["area"]
+    time_and_global_mean_value = weighted_mean(time_mean_value, area)
+    for variable in time_mean_value:
+        orig_unit = time_mean_value[variable].attrs["units"]
+        time_and_global_mean_value[variable].attrs["units"] = orig_unit
+    return time_and_global_mean_value
 
-    time_and_global_mean_bias = global_mean_bias.mean("time")
 
-    for variable in global_mean_bias:
-        orig_unit = global_mean_bias[variable].attrs["units"]
+@add_to_metrics("time_and_global_mean_bias")
+def time_and_global_mean_bias(diags):
```

Comment:
previously this metric was computed by taking a time-mean of the global-mean bias metric. Here we switch it to the global mean of the time-mean bias in order to be consistent with the diagnostic it is plotted with on the `maps.html` page. (In principle should be same value, but could be slight differences due to e.g. time sampling)